Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -160,6 +176,11 @@ - (void)timeout:(NSTimer *)timer
 
 #pragma mark - Public API
 
+RCT_EXPORT_METHOD(setRNConfiguration:(RCTLocationConfiguration)config)
```

Comment:
@javache "This method includes RN in the name to minimize the chances that the web spec will introduce an API with the same name."